nextcord/member.py

```diff
@@ -505,11 +505,14 @@ def mention(self) -> str:
     def display_name(self) -> str:
         """:class:`str`: Returns the user's display name.
 
-        For regular users this is just their username, but
-        if they have a guild specific nickname then that
-        is returned instead.
+        Returns the users displayed name.
+        So whatever is the first thing to exist from the following priority list:
```

Comment:
Something along the lines of
```suggestion
        This will return the name using the following hierachy:
```